upm-tools/AssemblyDefinitions/Bugsnag.asmdef

```diff
@@ -1,3 +1,3 @@
 ﻿{
```

Comment:
if you didnt already, please follow the steps in the CONTRIBUTING.md to build a local UPM package and check that this change doesnt break the package